### PR TITLE
feat(ui): Add try/catch for code splitting `@sentry/rrweb`

### PR DESCRIPTION
**Pull request by billyvg**
This will allow Sentry integrations to continue being loaded if `@sentry/rrweb` fails to load

### CODE REVIEW

File: src/sentry/static/sentry/app/bootstrap.tsx [hunk 1]
```diff
@@ -62,22 +62,29 @@ async function getSentryIntegrations(hasReplays: boolean = false) {
...
-    // should fix.
-    integrations.push(
-      new SentryRRWeb({
-        checkoutEveryNms: 60 * 1000, // 60 seconds
-      }) as any
-    );
+  if (hasReplays) {
+    try {
+      // eslint-disable-next-line no-console
+      console.log('[sentry] Instrumenting session with rrweb');
```
Comment (EvanPurkhiser) [inline src/sentry/static/sentry/app/bootstrap.tsx:69]: Is this message right? Isn't it more `Session instrumentation view enabled`

Comment (billyvg) [reply src/sentry/static/sentry/app/bootstrap.tsx:69]: It seems right, this block is only for rrweb

**Review by priscilawebdev — APPROVED** [review]

**Review by AbhiPrasad — APPROVED** [review]

**Comment by dcramer** [comment]
To me this change is unacceptable. There's not an issue in RRWeb, so who's giving me their phone number to wake them up at 2am when we've now broken this because _other changes_ we've made, and its no longer capturing replays? We wouldn't do this with our other integrations, we shouldn't be doing this here.

**Comment by billyvg** [comment]
@dcramer This can happen due to network issues and the bundle doesn't load at all and the alternative is that we do not have any Sentry integrations working at all. We can capture this in Sentry

**Comment by dcramer** [comment]
What network issue is causing RRWeb not to load but everything else is loading fine?

**Comment by billyvg** [comment]
`@sentry/rrweb` is split from the main bundle, there could be network issues that happen for `@sentry/rrweb` after the main bundle is loaded.

**Comment by billyvg** [comment]
Closing this... reverting the codesplit completely, see https://github.com/getsentry/sentry/pull/20413

**Pull request closed without merging**